cli/main.rs

```diff
@@ -123,103 +126,211 @@ pub fn print_file_info(worker: &Worker, url: &str) {
   }
 }
 
-fn main() {
-  #[cfg(windows)]
-  ansi_term::enable_ansi_support().ok(); // For Windows 10
+fn get_worker_and_state(
```

Comment:
“get” doesn’t sound right here since the worker and state are being constructed. I’d prefer “create”